blog/_posts/btcpay-server-2-0.md

```diff
@@ -0,0 +1,174 @@
+---
+title: "BTCPay Server 2.0: our biggest update yet!"
+date: 2024-10-29
+author: Pavlenex
+category:
+  - "releases"
+tags:
+  - "btcpay-server"
+  - "release"
+  - "2.0"
+coverImage: "/images/btcpay2-0-featured.png"
+---
+
+We are thrilled to announce the release of **BTCPay Server 2.0**, our most significant update since the project began in 2017! 
+
+This version is packed with new features, bug fixes, and crucial backend updates designed to set a solid foundation for the future and make BTCPay more powerful than ever before.
+
+BTCPay Server 2.0 realigns the internal architecture with the lessons we’ve learned since 2017, positioning us to better support both plugin developers and ourselves as we implement new features. While it may not seem immediately obvious, we expect users to see the full potential of 2.0 in the upcoming months, as the ecosystem continues to thrive and build on top of it.
+
+## 🔑 Key features
+
+- New interface localization
+- New sidebar-only navigation
+- New onboarding flow for new users and Point Of Sale
+- Three new e-commerce integrations (Wix, Odoo & BigCommerce)
+- Checkout v2 default (depreciation of the legacy checkout)
```

Comment:
it's not deprecation but removed / discarded?

Reply:
Fixed https://github.com/btcpayserver/blog/pull/61/commits/ec680a071e3a2030ffa566fcc3bc0aa352b133ea